src/utils.ts

```diff
@@ -98,12 +98,30 @@ export function getNpmConfig (runnerConfig: NpmConfigContainer) {
   if (runnerConfig.npm === undefined) {
     return {};
   }
-  return {
+  const cfg: { [key:string]: string | boolean | null } = {
     registry: runnerConfig.npm.registry || getDefaultRegistry(),
     'strict-ssl': runnerConfig.npm.strictSSL !== false,
     // Setting to false to avoid dealing with the generated file.
     'package-lock': runnerConfig.npm.packageLock === true
   };
+
+
+  // As npm config accepts only key-values pairs, we do the translation
+  if (runnerConfig.npm.scopedRegistries) {
+    for (const sr of runnerConfig.npm.scopedRegistries) {
+      cfg[`${sr.scope}:registry`] = sr.url;
+      if (sr.authToken) {
+        let fixedUrl = sr.url;
+        if (fixedUrl.startsWith('http://')) {
+          fixedUrl = sr.url.substring(5);
+        } else if (fixedUrl.startsWith('https://')) {
+          fixedUrl = sr.url.substring(6);
+        }
+        cfg[`${fixedUrl}`] = sr.authToken;
```

Comment:
Docs say you need to suffix with `:_authToken`

```suggestion
        cfg[`${fixedUrl}:_authToken`] = sr.authToken;
```